files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
+# window can be moved and resized with the operating system window
+# controls.  If this setting is false, the window has no operating
+# system border.  This setting has no effect if the "fullscreen"
+# setting in the Video section is true.  This setting can be toggled
+# in game using the "Window Border" button in the Video tab of the
+# Video panel in the Options menu.  It can also be toggled with the
+# "Window Border" check box in the OpenMW Launcher.
+window border = true
 
-[Saves]
-character =
-# Save when resting
-autosave = true
-# display time played
-timeplayed = false
+# This integer setting determines which screen the game will open on
+# in multi-monitor configurations.  This setting is particularly
+# important when "fullscreen" setting in the Video section is true,
+# since this is the only way to control which screen is used, but it
+# can also be used to control which screen a normal window or a
+# borderless window opens on as well.  This setting can be selected
+# from a pull down menu in the Graphics tab of the OpenMW Launcher,
+# but not adjusted during game play.
+screen = 0
 
-[Windows]
-inventory x = 0
-inventory y = 0.4275
-inventory w = 0.6225
-inventory h = 0.5725
+# Minimize the OpenMW window if it loses cursor focus.  This setting
+# has no effect if the "fullscreen" setting is false.  This setting is
+# primarily useful for single screen configurations, so that the
+# OpenMW screen in full screen mode can be minimized when the
+# operating system regains control of the mouse and keyboard.  On
+# multiple screen configurations, disabling this option make make it
+# easier to switch between screens while playing OpenMW.
+minimize on focus loss = true
 
-inventory container x = 0
-inventory container y = 0.4275
-inventory container w = 0.6225
-inventory container h = 0.5725
+# This boolean setting determines whether frame draws are synchronized
+# with the vertical refresh rate of your monitor.  Enabling this
+# setting can reduce "tearing", a visual defect caused by updating the
+# image buffer in the middle of a screen draw.  Enabling this option
+# typically implies limiting the framerate to 60 frames per second,
+# but may also introduce additional delays caused by having to wait
+# until the appropriate time (the vertical blanking interval) to draw
+# a frame.
+
+# This setting can be adjusted in game using the "VSync" button in the
+# Video tab of the Video panel in the Options menu.  It can also be
+# changed by toggling the "Vertical Sync" check box in the Graphics
+# tab of the OpenMW Launcher.
+vsync = false
 
-inventory barter x = 0
-inventory barter y = 0.4275
-inventory barter w = 0.6225
-inventory barter h = 0.5725
+# This floating point setting controls the gamma correction for all
+# video in the game.  This setting does not currently work under
+# Linux, and the in-game setting in the Options menu has been
+# disabled.
+gamma = 1.00
```

Comment:
Would it make sense to put this next to the 'contrast' option as it used to? These two options are kind of related.